gator/util.py

```diff
@@ -57,6 +66,18 @@ def get_symbol_answer(boolean_value):
     return "✘"
```

Comment:
These should be constants.

Reply:
Good idea! I fixed this and added some other useful constants as well. They are now used correctly in the `util` module. There are test cases for these new constants in `test_constants. I can confirm that these new constants are not changing the output in any way.

I'm going to mark this issue as resolved.